### ISSUE TITLE
Poll questions are not accessible via keyboard

### ISSUE THREAD

**Issue by dariiing** [xwiki-labs/cryptpad#1966]
On forms, these type of questions are not reachable via keyboard
![Image](https://github.com/user-attachments/assets/8b8e5a0c-a243-422c-9e37-a815a3298fe1)
